### ISSUE TITLE
Can't clone Jekyll blog site from set up

### ISSUE THREAD

**Issue by charlottebrf** [charlottebrf/charlottebrf.github.com#1]
When copying & pasting the "Install Jekyll-Boostrap" it doesn't install on terminal- see screenshot:

![jekyll](https://cloud.githubusercontent.com/assets/16884984/16079244/d5f3f898-3302-11e6-9dcb-4a29a6d4061f.png)

**Comment by eddiejaoude** [comment]
Can you copy & paste your terminal output?

**Comment by eddiejaoude** [comment]
Run the commands individually and then if any fail, we can see which one it failed on

eg...
1. `git clone https://github.com/plusjade/jekyll-bootstrap.git charlottebrf.github.com`
2. `cd charlottebrf.github.com` 
3. `git remote set-url origin git@github.com:charlottebrf/charlottebrf.github.com.git`
4. `git push origin master`

**Comment by eddiejaoude** [comment]
Yep, the command they supply is one long line & should be broken down by a comma or each command run separately.

I have broken it for you here:
1. `git clone https://github.com/plusjade/jekyll-bootstrap.git charlottebrf.github.com`
2. `cd charlottebrf.github.com` 
3. `git remote set-url origin git@github.com:charlottebrf/charlottebrf.github.com.git`
4. `git push origin master`

**Comment by charlottebrf** [comment]
Tried the second way & got this: 

Charlottes-MacBook-Air-2:GitHub charlottefereday$ git clone https://github.com/plusjade/jekyll-bootstrap.git charlottebrf.github.com
Cloning into 'charlottebrf.github.com'...
remote: Counting objects: 2333, done.
remote: Total 2333 (delta 0), reused 0 (delta 0), pack-reused 2333
Receiving objects: 100% (2333/2333), 825.71 KiB | 147.00 KiB/s, done.
Resolving deltas: 100% (988/988), done.
Checking connectivity... done.
Charlottes-MacBook-Air-2:GitHub charlottefereday$ cd charlottebrf.github.com
Charlottes-MacBook-Air-2:charlottebrf.github.com charlottefereday$ 
Charlottes-MacBook-Air-2:charlottebrf.github.com charlottefereday$ git remote set-url origin git@github.com:charlottebrf/charlottebrf.github.com.git
Charlottes-MacBook-Air-2:charlottebrf.github.com charlottefereday$ git push origin master
Warning: Permanently added the RSA host key for IP address '192.30.252.121' to the list of known hosts.
Permission denied (publickey).
fatal: Could not read from remote repository.

**Comment by charlottebrf** [comment]
I think that the issue is with the last command?

**Comment by eddiejaoude** [comment]
Can you run this command `git remote -v`

**Comment by charlottebrf** [comment]
I've realised it's because I don't have an SSH key setup... so I'm working
on that then hopefully should be able to run commands

On 16 June 2016 at 08:34, Eddie Jaoude <notifications@github.com> wrote:

> Can you run this command git remote -v
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/charlottebrf/charlottebrf.github.com/issues/1#issuecomment-226400024>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AQGk-CU94-r4Kd2pa921Ot5a433UgWKNks5qMO6DgaJpZM4I2SvF>
> .
>



-- 
---------------------------------------------------------------------------

*Charlotte Fereday*

Programmes Manager @codefirstgirls

Tweeting & retweeting @charlottebrf

*Mainly: Tech/Education/Hispanic studies *

**Comment by eddiejaoude** [comment]
👍 

FYI: https://help.github.com/articles/generating-an-ssh-key/

**Comment by charlottebrf** [comment]
Thanks so much Eddie 😀😀

On 16 June 2016 at 08:42, Eddie Jaoude <notifications@github.com> wrote:

> 👍
>
> FYI: https://help.github.com/articles/generating-an-ssh-key/
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/charlottebrf/charlottebrf.github.com/issues/1#issuecomment-226401045>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AQGk-GwvnQun9h0jh047g11zZvSyJlMcks5qMPBAgaJpZM4I2SvF>
> .
>



-- 
---------------------------------------------------------------------------

*Charlotte Fereday*

Programmes Manager @codefirstgirls

Tweeting & retweeting @charlottebrf

*Mainly: Tech/Education/Hispanic studies *
